docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -155,49 +168,9 @@ Preparing For a Release
 #. Update the staging documentation.
    (For a documentation release ``${VERSION}`` should be the base release version in this step).
```

Comment:
This is unchanged here but I'm looking through the release docs and want to confirm - this means that if `$VERSION` is set as `0.3.1+doc1` then the engineer should now run `export VERSION=0.3.1`. If so I can make an issue to automate this.